packages/react/src/components/Pagination/PaginationRoot.tsx

```diff
@@ -14,6 +14,10 @@ export const PaginationContext = createContext<PaginationContextProps>({
 });
 
 export type PaginationRootProps = {
+  /**
+   * Sets screen reader label for the pagination area
+   */
+  'aria-label': string;
```

Comment:
```suggestion
```
Lets do a default instead

Reply:
Ah! Good catch, did not remember this was what we concluded in Breadcrumbs, I'll fix <3